packages/common/src/ContractInteractor.ts

```diff
@@ -363,6 +377,17 @@ export class ContractInteractor {
     return latestBlock.gasLimit
   }
 
+  _fixGasFees (relayRequest: RelayRequest): { gasPrice?: string, maxFeePerGas?: string, maxPriorityFeePerGas?: string } {
```

Comment:
There is `RelayProvider::_fixGasFees` and `ContractInteractor::_fixGasFees` that do similar but not exactly same things. I suggest having two different names :-)

Reply:
Yes, they are similar in nature but different in impl, but they are both "internal" functions, so I guess having the same name is ok, no?